README.md

```diff
@@ -1,24 +1,26 @@
 # Scholarx-frontend
+
 Frontend dashboard of the scholarX platform
 
 ## Setup Development Environment
 
 ### **Project setup walkthrough :- https://youtu.be/1STopJMM2nM**
 
 1. Clone your forked repository
-    ```
-    git clone https://github.com/USERNAME/scholarx-frontend
-    cd scholarx-frontend
-    ```
+   ```
+   git clone https://github.com/sef-global/scholarx-frontend.git
```

Comment:
Actually this needs to be username since we are cloning the fork.
```suggestion
   git clone https://github.com/YOUR_USERNAME/scholarx-frontend.git
```